benchmarks/pom.xml

```diff
@@ -200,23 +200,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
-        <version>2.6</version>
+        <version>3.2.0</version>
```

Comment:
the version is 3.2.0 not the 3.3.0 you mentioned in the description of this PR. which one should it be?

Reply:
It is 3.2.0. The commit message was a typo.